lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -55,26 +100,99 @@ module Resources
+          }
+        },
+        {
+          'id': 'CI-490153231-4',
+          'use': 'usual',
+          'type': {
+            'coding': [
+              {
+                'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4',
+                'code': '2553236785',
+                'display': 'MIllennium Person Identifier',
+                'userSelected': true
+              }
+            ],
+            'text': 'MIllennium Person Identifier'
+          },
+          'value': '12868',
           'period': {
-            'start': '2020-06-30T20:26:38.000Z'
+            'start': '2020-12-18T19:15:06.000Z'
           }
         }
       ],
       'active': true,
       'patient': {
-        'reference': 'Patient/12724066',
-        'display': 'SMART, NANCY'
+        'reference': 'Patient/12724068',
+        'display': 'SMART, HAILEY'
       },
       'relationship': [
         {
-          'id': 'CI-490017058-0-0',
+          'id': 'CI-490017041-0',
           'extension': [
             {
               'valuePeriod': {
                 'start': '2019-12-26T16:06:26.000Z'
               },
               'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            },
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                    'code': '156'
+                  },
+                  {
+                    'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+                    'code': 'MTH',
+                    'display': 'Mother',
+                    'userSelected': false
+                  }
+                ]
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relation'
+            }
+          ],
+          'coding': [
+            {
+              'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/351',
+              'code': '1150',
+              'display': 'Default Guarantor',
+              'userSelected': true
+            },
+            {
+              'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+              'code': 'GT',
+              'display': 'Guarantor',
+              'userSelected': false
+            }
+          ],
+          'text': 'Default Guarantor'
+        },
+        {
+          'id': 'CI-490017055-1',
+          'extension': [
+            {
+              'valuePeriod': {
+                'start': '2019-12-26T16:06:26.000Z'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            },
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                    'code': '152'
```

Comment:
Missing standard code and display and text like above.

Reply:
added here https://github.com/cerner/fhir.cerner.com/pull/644/commits/8826890486713f37b02cddcff2f7f3870f42cb16#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR199-R204
https://github.com/cerner/fhir.cerner.com/pull/644/commits/8826890486713f37b02cddcff2f7f3870f42cb16#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR194-R196

Reply:
+1